tools/README.rst

```diff
@@ -0,0 +1,305 @@
+
+Providing an HawkId::
+
+    $ node get_hawk_user_info.js 88d5a28f545bb406ddc6c6a5276cbfe0aa10fdba425f4808e2d6c3acdbfdaeda
+    Trying with HawkIdHmac: de9cd5c5ded9e2df982723d96361f56c0d72c936dc177cbff1f147bac1445f63
+    { anonymous: false, userId: 'foobar@example.com' }
+
+Providing an HawkIdHmac::
+
+    $ nodeget_hawk_user_info.js de9cd5c5ded9e2df982723d96361f56c0d72c936dc177cbff1f147bac1445f63
+    Trying with HawkIdHmac: dcf3932ac6c0ed48994bb17c5ecc150e03e84a76e523b698c8cc75c2ca278611
+    Trying with HawkIdHmac: de9cd5c5ded9e2df982723d96361f56c0d72c936dc177cbff1f147bac1445f63
+    { anonymous: false, userId: '<ciphered>' }
+
+Providing an unauthenticated HawkIdHmac::
+
+    $ nodeget_hawk_user_info.js 81d2afea33181e32023c9042b42157ebf453d3c04435b386ded7c378fb338b01
+    Trying with HawkIdHmac: c4c9a59a1a12719e395cb64e35d53d515335612e4b3208c51c89beecaa496393
+    Trying with HawkIdHmac: 81d2afea33181e32023c9042b42157ebf453d3c04435b386ded7c378fb338b01
+    { anonymous: true }
+
+
+Redis Usage
+-----------
+
+- ``get_redis_usage.js``
+
+This script gives you general information about the redis keys::
+
+    # Server
+    [...]
+    
+    # Clients
+    [...]
+    
+    # Memory
+    [...]
+
+    # Persistence
+    [...]
+    
+    # Stats
+    [...]
+    
+    # Replication
+    [...]
+    
+    # CPU
+    [...]
+    
+    # Keyspace
+    db0:keys=179,expires=58,avg_ttl=2118094581
+    
+     ==== 
+    
+    spurls.*: 	64
+    spurls.6e0a93dd218b767f799be64534c01c1f0706361a6b0caba1ca9c8099d2d8078b.6e0a93dd218b767f799be64534c01c1f0706361a6b0caba1ca9c8099d2d8078b
+    spurls.a33b8202d462bbfa0bf1559b8ff3e05f710832c5103a142a2263e178810f858f
+
+    callurl.*: 	22
+    callurl.we8ADTMY6o8
+    callurl.SPwwEPBW7OA
+
+    userUrls.*: 	21
+    userUrls.40057524c466604ecad39c88871a896dee5fd4718cd37373f4703db12fbd5ee7
+    userUrls.24ce5f27583b5eb2de9655c21a221546e97629e892a871e161ebdab861317829
+
+    call.*: 	0
+
+    userCalls.*: 	18
+    userCalls.055620865c42a71a1049d75692411095d9d68ba0843ff4c8a8fc825643c0756e
+    userCalls.23cf69cbd9265e9b78444f71c43beee6d7f85976df284af575d5c37d4cf780f6
+
+    callstate.*: 	0
+
+    hawkuser.*: 	1
+    hawkuser.de9cd5c5ded9e2df982723d96361f56c0d72c936dc177cbff1f147bac1445f63
+
+    userid.*: 	1
+    userid.de9cd5c5ded9e2df982723d96361f56c0d72c936dc177cbff1f147bac1445f63
+
+    hawk.*: 	33
+    hawk.fabaf4f9f60c6f8d97158c75f0b9b2661738130eb654eed13d5ecdc8739d0f1a
+    hawk.23cf69cbd9265e9b78444f71c43beee6d7f85976df284af575d5c37d4cf780f6
+
+    oauth.token.*: 	1
+    oauth.token.de9cd5c5ded9e2df982723d96361f56c0d72c936dc177cbff1f147bac1445f63
+
+    oauth.state.*: 	1
+    oauth.state.de9cd5c5ded9e2df982723d96361f56c0d72c936dc177cbff1f147bac1445f63
+
+    userRooms.*: 	4
+    userRooms.b8ae434636685b6d31c0b0efb96e649bd67c33c1c3fa9a23caaf3aaf804cfdd9
+    userRooms.494e14e5f507317b7392eafb3ca2a2372bd61a5735dbc06d9d70abe74b7d1d57
+
+    rooms.*: 	0
+
+
+Remove OLD keys
+---------------
+
+- ``remove_old_keys.js``
```

Comment:
Can we specify a duration here? That could be useful to remove keys older than, for instance, the duration of a session.